src/coreclr-embedding-test/src/ScriptingManager.hpp

```diff
@@ -0,0 +1,45 @@
+//
+//  ScriptingManager.hpp
+//  embedding_test
+//
+//  Created by Leonidas Neftali Gonzalez Campos on 06/12/23.
+//
+#pragma once
+#include <dlfcn.h>
```

Comment:
Same as with dlsym

Reply:
Will include a conditional check for the OS